src/toil_scripts/gatk_germline/germline_config.py

```diff
@@ -8,79 +8,97 @@ def generate_config():
         # Edit the values in this configuration file and then rerun the pipeline
         # Comments (beginning with #) do not need to be removed. Optional parameters may be left blank.
         ##############################################################################################################
-        # Required: Reference Genome URL (hg19.fa)
+        # Required: Reference Genome URL
```

Comment:
All of the parameter docs in this config file need serious love. E.g., instead of:

`# Required: Reference Genome URL`

write:

`# Required: URL or local path to reference genome FASTA file`